src/pages/edge/adobe-journey-optimizer-decisioning/review-simulate.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Adobe Experience Platform Assurance Review & Simulate
+description: This is a feature of the Adobe Experience Platform Assurance which allows the users to review the content sent to devices for each request and simulate different experiences on the same devices in real time.
+keywords:
+- Adobe Experience Platform Assurance
+- Review & Simulate Feature Introduction
+---
+
+# Adobe Experience Platform Assurance Review & Simulate
```

Comment:
Please keep the headings consistent. You can call it Decisioning Review & Simulate view